WorkloadManagementSystem/Service/MatcherHandler.py

```diff
@@ -117,7 +117,9 @@ def export_getMatchingTaskQueues( self, resourceDict ):
       negativeCond = self.limiter.getNegativeCondForSite( resourceDict[ 'Site' ] )
     else:
       negativeCond = self.limiter.getNegativeCond()
-    return gTaskQueueDB.retrieveTaskQueuesThatMatch( resourceDict, negativeCond = negativeCond )
+    matcher = Matcher()
```

Comment:
Wouldn't it be faster to initialize the Matcher object like in `export_requestJob`?